src/content/self-hosted/install/backup-restore.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Backup and Restore
+description: Backup and Restore your Okteto instance
+id: backup
+sidebar_label: Backup and Restore
+---
+
+Backup and restore planning is essential to ensure the availability and integrity of your Okteto instance in case of unexpected failures or disaster recovery.
+This guide outlines the steps to recover an Okteto instance in such scenarios.
```

Comment:
```suggestion
This guide outlines the steps to recover an Okteto instance in the event of an unexpected failure and for disaster recovery preparedness.
```

I think we can condense these first two sentences into this one. WDYT @pchico83?